src/FileProcessor/TypoScript/TypoScriptFileProcessor.php

```diff
@@ -81,9 +82,16 @@ public function getSupportedFileExtensions(): array
         return $this->allowedFileExtensions;
     }
 
+    /**
+     * @param mixed[] $configuration
+     */
     public function configure(array $configuration): void
     {
-        $this->allowedFileExtensions = $configuration[self::ALLOWED_FILE_EXTENSIONS] ?? [];
+        $allowedFileExtensions = $configuration[self::ALLOWED_FILE_EXTENSIONS] ?? $configuration;
+        Assert::isArray($allowedFileExtensions);
+        Assert::allString($allowedFileExtensions);
+
+        $this->allowedFileExtensions = $allowedFileExtensions;
```

Comment:
It's a good practise to validate user's input - if it's a form, an API or configuration in config.
Here we make sure we fail as soon as `configure()` is called with invalid input and let use know to correct it :+1: